static/js/util/api.js

```diff
@@ -1,4 +1,4 @@
-// @flow
+// @flow weak
```

Comment:
My main concern is that these values can be null and if there are type errors due to a null value, they might be important to handle. I agree that it complicates type checking. Maybe we can convert null values to empty strings when we receive the profile from the API, then we should be safe in assuming no null values